### DISCUSSION TITLE
After uploading to Media, i got “empty” image symbols

### DISCUSSION THREAD

**Discussion by Kennypurnomo** [Interfacelab/ilab-media-tools#222] [Q&A]
After uploading to Media, i got “empty” image symbols. I have managed my billing section in Google Cloud Console. What might be the problem? Is it Wordfence?

Thank you

https://img.sharemyimage.com/2023/09/21/Screenshot-2023-09-21-204610.png

**Comment by jawngee** [comment]
Billing section of google?

The most likely case is that you have permissions setup wrong on your bucket.  If you see an empty square with a cloud icon on it in the media library (in grid mode) then that is probably what is happening.

**Reply by Kennypurnomo** [reply]
Yes billing section of google. i am using Wordfence. Do you think it is the reason? If you have any experience with it, how do i setup wordfence so that i can also use Media Cloud?

**Comment by jawngee** [comment]
I don't understand what the "billing section of google" means.

It's not WordFence, you don't have Google Cloud Storage configured properly.  It's a permissions issue.  If you look in the web inspector do you see 403 errors for the images?
